src/cli/backup/exchange_integration_test.go

```diff
@@ -396,8 +397,9 @@ func (suite *PreparedBackupExchangeIntegrationSuite) TestExchangeDetailsCmd() {
 			bID := suite.backupOps[set]
 
 			// fetch the details from the repo first
-			deets, _, err := suite.repo.BackupDetails(ctx, string(bID))
-			require.NoError(t, err)
+			deets, _, errs := suite.repo.BackupDetails(ctx, string(bID))
+			require.NoError(t, errs.Err())
+			require.Empty(t, errs.Errs())
```

Comment:
Should this be `deets` also?

Reply:
I think deets gets used below, and is therefore checked in that way.